test/lib/cql_test_env.cc

```diff
@@ -675,6 +675,7 @@ class single_node_cql_env : public cql_test_env {
             gcfg.cluster_name = "Test Cluster";
             gcfg.seeds = std::move(seeds);
             gcfg.skip_wait_for_gossip_to_settle = 0;
+            gcfg.shutdown_announce_ms = 0;
```

Comment:
Amazing!

While I kept optimizing test/cql-pytest/run to get a test running and shutting down in 0.3 seconds, I forgot that this should also be done in the boost tests... Imagine how many seconds we wasted in the last 6 years running those tests :-(